.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.root": "/dist"
```

Comment:
I think we should ignore adding specific VSCode settings? Might want to ignore `.vscode` in `.gitignore`.

Reply:
I removed the file in the next commit.